styles/typography.css

```diff
@@ -35,9 +35,19 @@
   .prose dd {
     @apply ml-8;
   }
-  .prose p img {
+  /* Keep inline images inside text flow from forcing line breaks */
+  .article p img,
+  .article table img,
+  .article dd img,
+  .prose p img,
+  .prose table img,
+  .prose dd img {
     margin: 0;
     display: inline-block;
+    vertical-align: text-bottom;
```

Comment:
Do we want `text-bottom` or `baseline`?

https://www.w3schools.com/cssref/tryit.php?filename=trycss_vertical-align

Reply:
hmmm maybe I should just remove this and not be opinionated about it until we see what behavior people expect?